config-templates/redhat/odf-local/changelog.json

```diff
@@ -4,14 +4,15 @@
             "status": "supported",
             "latest-revision": {
                 "revision": "1",
-                "release-date": "05 June 2025",
+                "release-date": "24 June 2025",
                 "cves": "",
-                "golang": "1.23.9",
+                "golang": "1.23.10",
                 "ubi": "",
                 "new-features": [
-                    "ODF 4.18 initial release"
+                    "ODF 4.18 release"
                 ],
                 "fixes": [
+                    "Latest bug fixes and minor improvements"
```

Comment:
Since this is first ODF 4.18 release for customers. So lets remove `"Latest bug fixes and minor improvements"`